widget/playground/src/constants/presets.ts

```diff
@@ -281,7 +164,8 @@ export const WIDGET_COLORS: { key: WidgetColorsKeys; label: string }[] = [
     label: 'Foreground',
   },
 ];
-export const DEFAULT_COLORS = {
+
+export const DEFAULT_THEME = {
```

Comment:
theme includes other properties like `spacing`, ... . So please rename this to `DEFAULT_THEME_COLORS`